2048/2048/src/main/java/com/appcoins/eskills2048/MainView.java

```diff
@@ -75,10 +78,12 @@
     private int titleWidthOpponentRank;
     private int titleWidthOpponentName;
     private int titleWidthOpponentScore;
+    private final Context context;
```

Comment:
you can use the `getContext` method instead of managing the context yourself

Reply:
In order to build a View you always need some sort of context, there is no problem with it so the constructor is ok. The rest, you can clean it